src/blocking/dac.rs

```diff
@@ -0,0 +1,11 @@
+//! Digital-to-analog conversion traits
+
+/// A trait used to identify a digital-to-analog converter, and its
+/// most fundamental features.
```

Comment:
```suggestion
//! Trait representing a single-channel digital-to-analog converter (DAC).
```
This was probably reverted during merge.